docs/devops-guide/docker.md

```diff
@@ -6,15 +6,18 @@ sidebar_label: Docker
 
 ## Quick start
 
-In order to quickly run Jitsi Meet on a machine running Docker and Docker Compose,
-follow these steps:
+In order to quickly run Jitsi Meet on a machine running Docker, make use of Docker Compose and follow these steps:
 
 * Download and extract the [latest release]
+  * `wget $(curl -s https://api.github.com/repos/jitsi/docker-jitsi-meet/releases/latest | grep 'zip' | cut -d\" -f4)` (change zip to tar.gz if you prefer)
```

Comment:
I think this is a bit too specific, maybe at some point urls or commands will change etc., so I guess every user, especially someone you wants to run a server, should be able to download and extract a file.

Reply:
Hi @toby63, thanks for the review.
A year or three ago I was completely new to self-hosting stuff. Now I got Nextcloud and Funkwhale running. Still far from an expert, and completely new to Docker. Especially in that beginning phase these concrete explanations help you get started. Rather than studying what might be the best commands to know, you take a project and see if you can get it up. You learn about wget and curl in the process. I love documentation that supports this process, as it helped me a lot.
From this perspective, I also made this adjustment. Sure self-hosters should know this, but this might just be the reminder or starter they need to get to the next level.
Moreover, the next step _is_ explained/specified with code (specifying the cp command). For me it felt like this first step was missing that specification.

Reply:
If you want to give more specific instructions, I suggest to give them one after another, like: 
1. wget for download
2. and then unzip (or something similar) for extraction
3. then cd into the folder

This way something can be learned, but with easier and structured commands.